quaistats/quaistats.go

```diff
@@ -828,10 +823,82 @@ func (s *Service) reportStats(conn *connWrapper) error {
 			GasPrice: gasprice,
 			Syncing:  syncing,
 			Uptime:   100,
+			Chain: common.NodeLocation.Name(),
 		},
 	}
 	report := map[string][]interface{}{
 		"emit": {"stats", stats},
 	}
 	return conn.WriteJSON(report)
 }
+
+
+// peerStats is the information to report about peers.
+type peerStats struct {
+	Chain string `json:"chain"`
+	Count int `json:"count"`
+	PeerData []*peerData `json:"peerData"`
+}
+
+// peerStat is the information to report about peers.
+type peerData struct {
+	Chain 			string `json:"chain"`
+	Enode      		string `json:"enode"` // Unique node identifier (enode URL)
+	SoftwareName    string `json:"softwareName"` // Node software version
+	LocalAddress  	string `json:"localAddress"`
+	RemoteAddress 	string `json:"remoteAddress"`
+	RTT 			string `json:"rtt"`
+	LatestHeight	uint64 `json:"latestHeight"`
+	LatestHash		string `json:"latestHash"`
+	// TODO
+	RecvLastBlockTime time.Time `json:"recvLastBlockTime"`
+	Uptime			string `json:"uptime"`
+	PeerUptime		string `json:"peerUptime"`
```

Comment:
Lets rename this to `ConnectedTime` since all of this data is peer data. It would be confusing to see `peer.Uptime` and `peer.PeerUptime` side by side

Reply:
Fixed to `ConnectTime`. Need to do some digging on `PeerUptime` since that isn't something natively stored in the peer.